avocado/test.py

```diff
@@ -11,7 +11,6 @@
 # This code was inspired in the autotest project,
 # client/shared/test.py
 # Authors: Martin J Bligh <mbligh@google.com>, Andy Whitcroft <apw@shadowen.org>
-
```

Comment:
Sorry, it was reformated automatically by Eclipse... I'll change it in the next version.